internal/jobcontainers/jobcontainer.go

```diff
@@ -166,10 +201,29 @@ func (c *JobContainer) CreateProcess(ctx context.Context, config interface{}) (_
 		return nil, errors.New("unsupported process config passed in")
 	}
 
-	// Replace any occurrences of the sandbox mount point env variable in the commandline.
-	// %CONTAINER_SANDBOX_MOUNTPOINT%\mybinary.exe -> C:\C\123456789\mybinary.exe
+	// Replace any occurences of the sandbox mount env variable in the commandline.
+	// For example: %CONTAINER_SANDBOX_MOUNTPOINT%\mybinary.exe -> C:\<rootfslocation>\mybinary.exe.
 	commandLine, _ := c.replaceWithMountPoint(conf.CommandLine)
```

Comment:
Can we get away from doing the `CONTAINER_SANDBOX_MOUNTPOINT` hacks on OS versions where we have proper binding support? It would be nice to deprecate that.

Reply:
I think the plan was to leave it for now for backwards compat on builds that didn't have it, so you could upgrade from 2019 -> 2022 and not have to change anything. cc @marosset

Reply:
I agree though, probably further down the line we can get rid of this for bindflt machines